src/Logic/Utilities.cs

```diff
@@ -3473,5 +3220,35 @@ public static int GetNumberOfLines(string text)
             return lines;
         }
 
+        public static string FixInvalidSsaTags(string text)
+        {
+            if (text == null || text.Length <= 6)
+                return text;
+
+            text = text.Replace("{ \\a", "{\\a");
+            text = text.Replace("{\\a ", "{\\a");
+            text = text.Replace("{\\an ", "{\\an");
+
+            var pre = string.Empty;
+            var idx = text.IndexOf("{\\a", StringComparison.Ordinal);
+            var firstTime = idx;
+            while (idx >= 0)
+            {
+                var endIdx = text.IndexOf('}', idx + 2);
+                if (endIdx < idx)
+                    break;
+                if (firstTime == idx)
+                    pre = text.Substring(idx, endIdx - idx + 1);

```

Comment:
Most player only looks for one Ssa tag all the others will be ignore, so move it at the beginning